generators/languages/generator.mjs

```diff
@@ -21,16 +21,19 @@ import chalk from 'chalk';
 import _ from 'lodash';
 
 import BaseApplicationGenerator from '../base-application/index.mjs';
-import { SERVER_TEST_SRC_DIR, SERVER_MAIN_RES_DIR, SERVER_TEST_RES_DIR } from '../generator-constants.mjs';
 import { askForLanguages, askI18n } from './prompts.mjs';
 import statistics from '../statistics.cjs';
 
 import { GENERATOR_LANGUAGES, GENERATOR_BOOTSTRAP_APPLICATION } from '../generator-list.mjs';
 import { clientI18nFiles } from './files.mjs';
 import { writeEntityFiles } from './entity-files.mjs';
-import { languageToJavaLanguage } from './utils.mjs';
-import { replaceContent } from '../utils.mjs';
 import TranslationData from './translation-data.mjs';
+import { findLanguageForTag, supportedLanguages } from './support/languages.mjs';
+import { updateLanguages as updateLanguagesInAngular } from '../angular/support/index.mjs';
```

Comment:
Won't it be better to get the client (or any other related) technology to compose the language generator instead?

Reply:
The technology generators are already composing with languages:
https://github.com/jhipster/generator-jhipster/blob/e7c95e52a6ba86b6a148a812d17fc6f6978bb38d/generators/angular/generator.mjs#L49-L55

This PR is moving the needles implementation there.

Reply:
Thank you for your answer

I may have misspoken, I was trying to say that import from `angular` to `languages` looks normal (meaning you want to generate the i18n for your language) but the reverse looks less solid: we would be forced to introduce coupling from the language technology generator each time we want to introduce a new one (i.e. quarkus, svelte...) instead of 'just' implementing the language templates and abstract methods at the technology generator level

Reply:
The languages generator have a particular behavior, it's possible to add a language to an already existing project.
It's possible for the blueprint to ignore this incremental behavior an iterate over the configured languages.
Unless we remove this incremental behavior or do some more advanced options passing (add something like --single-entity to generate a single language and ignore everything else) we should keep the languages generator like this.

This is the reason https://github.com/jhipster/generator-jhipster/issues/19114 will not be implemented.